example/conf.py

```diff
@@ -61,7 +61,6 @@
 # The full version, including alpha/beta/rc tags.
 version = '7.4.3-dev'
 
-
```

Comment:
Two lines before new functions is PEP8-style, so there was no need to remove it.
Let's not worry right now, but I'll find some time to add linting and formatting to this repo in the future.